source/_integrations/generic.markdown

```diff
@@ -25,26 +25,23 @@ You must enter a URL in at least one of the fields **Still Image URL** or **Stre
 
 [Templates](/topics/templating/) are allowed in the URL fields, which can be used to select different images or parameterize the URL depending on the status of sensors.  Template validity and network access are checked during the configuration steps.
 
-{% configuration_basic %}
-still_image_url:
-  description: "The URL your camera serves the image on, e.g., `http://192.168.1.21:2112/`. Can be a [template](/topics/templating/). At least one of still_image_url or stream_source must be provided."
-stream_source:
-  description: "The URL your camera serves the live stream on, e.g., `rtsp://user:pass@192.168.1.21:554/`. Can be a [template](/topics/templating/). Usernames and passwords must be embedded in the URL. At least one of still_image_url or stream_source must be provided. Note that a stream_source without a still_image_url can only be used if the [stream integration](/integrations/stream/) is configured."
-username:
-  description: The username for accessing your camera. Note that this username applies only to still_image_url and not to stream_source.
-password:
-  description: The password for accessing your camera. Note that this password applies only to still_image_url and not to stream_source.
-authentication:
-  description: "Type for authenticating the requests `basic` or `digest`."
-limit_refetch_to_url_change:
-  description: Limits re-fetching of the remote image to when the URL changes. Only relevant if using a template to fetch the remote image.
-framerate:
-  description: The number of frames-per-second (FPS) of the stream. Can cause heavy traffic on the network and/or heavy load on the camera.
-verify_ssl:
-  description: Enable or disable SSL certificate verification. Set to false to use an http-only camera, or you have a self-signed SSL certificate and haven't installed the CA certificate to enable verification.
-rtsp_transport:
-  description: "Set the RTSP transport protocol to `tcp`, `udp`, `udp_multicast` or `http`."
-{% endconfiguration_basic %}
+### Generic Camera Configuration
+
+You can configure additional Generic Camera options for each camera through the configuration options in the GUI. The configure button can be found after clicking into the camera name under Generic Camera on the integration details page.
+
+| Option | Description |
+| -------| ----------- |
```

Comment:
We prefer to avoid tables and use configuration_basic.

Please revert that change.

Reply:
Sorry, the usage was a little unclear from the developer docs so I was copying the usage from another integration (ONVIF). Have reverted that now.